includes/class-wc-payments-order-service.php

```diff
@@ -773,12 +773,12 @@ public function get_fraud_meta_box_type_for_order( $order ) : string {
 	/**
 	 * Given the payment intent data, adds it to the given order as metadata and parses any notes that need to be added
 	 *
-	 * @param WC_Order                           $order The order.
-	 * @param WC_Payments_API_Abstract_Intention $intent The payment or setup intention object.
+	 * @param WC_Order                                                          $order The order.
+	 * @param WC_Payments_API_Payment_Intention|WC_Payments_API_Setup_Intention $intent The payment or setup intention object.
```

Comment:
I was getting Psalm UndefinedMethod linting error, so replaced the abstract class type hint with a list of possible concrete types. I wonder if there's a proper way to use abstract types as type hints without Psalm complaining.